source/PlayerInfo.cpp

```diff
@@ -1263,7 +1288,7 @@ void PlayerInfo::Land(UI *ui)
 	else if(ui && !inactiveMissions.empty())
 	{
 		string message = "These active missions or jobs were deactivated due to a missing definition"
-			" - perhaps you recently removed a plugin?\n";
+		" - perhaps you recently removed a plugin?\n";
```

Comment:
This change seems unrelated to this PR. (And also seems not needed.)

Reply:
done, with next commit